python/lsst/ci/eups_module.py

```diff
@@ -0,0 +1,84 @@
+import asyncio
+import os
+from typing import List
+
+import re
+
+import eups  # type: ignore
+import eups.tags  # type: ignore
+
+from lsst.ci.git import Git
+
+
+class EupsModule:
```

Comment:
Please document this class and its constructor arguments here.

Please document all methods.